data/assets/examples/timeframe/simple.asset

```diff
@@ -0,0 +1,37 @@
+-- This example creates a union out of two simpler TimeFrameIntervals and uses it for a
+-- SceneGraphNode. The first TimeFrameInterval covers January 1st, 2000 and the second
+-- TimeFrameInterval covers March 1st, 2002. The resulting TimeFrameUnion will cover both
+-- January 1st, 2000 and March 1st, 2002.
+
+local Node = {
+  Identifier = "TimeFrameUnion_Example_Simple",
+  TimeFrame = {
```

Comment:
Would it make sense to have a renderable that is visible during this timeframe?  Right now it's really hard to try this example out since nothing is visible